_articles/getting-paid.md

```diff
@@ -17,7 +17,7 @@ related:
 
 ## Why some people seek financial support
 
-Much of open source work is volunteered. For example, someone might come across a bug in a project they use and submit a quick fix, or they might enjoy tinkering with an open source project in their spare time.
+Much of the open source work is volunteered. For example, someone might come across a bug in a project they use and submit a quick fix, or they might enjoy tinkering with an open source project in their spare time.
```

Comment:
This still doesn't read quite right to me. How about:

```suggestion
Much of the work of open source is volunteer. For example, someone might come across a bug in a project they use and submit a quick fix, or they might enjoy tinkering with an open source project in their spare time.
```

Reply:
How about, "_Much of the open source work is voluntary_" or "_Much of the work of open source is voluntary_"?

Reply:
Okay, then I'll reflect on this change. Thanks for helping!

Reply:
I,'m new